src/js/components/Table/Paginated/PaginatedTable.tsx

```diff
@@ -63,6 +63,7 @@ const propTypes = {
   rowsAreSelectable: PropTypes.bool,
   searchPlaceholder: PropTypes.string,
   defaultSort: PropTypes.string,
+  placeholder: PropTypes.string,
```

Comment:
I would suggest to change the param name to be something more descriptive than `placeholder` such as `noResultsText`.